clar2wasm/src/words/mod.rs

```diff
@@ -158,36 +133,91 @@ pub(crate) static WORDS: &[&'static dyn Word] = &[
     &tuples::TupleMerge,
 ];
 
-pub trait Word: Sync + core::fmt::Debug {
+pub trait SimpleWord: Sync + core::fmt::Debug {
     fn name(&self) -> ClarityName;
 
     fn traverse(
```

Comment:
I think this should be named `visit`.

Reply:
I'm willing to yield on the other discussion for now if we at least rename this method to `visit`.

Reply:
done!